packages/shared_preferences/shared_preferences/example/macos/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Flutter-related
```

Comment:
nit: This and the other .gitignores can be removed.

Reply:
Done